tests/secc/states/test_iso15118_2_states.py

```diff
@@ -193,6 +194,23 @@ async def test_authorization_next_state_on_authorization_request(
             == expected_evse_processing
         )
 
+    async def test_authorization_req_gen_challange(self):
+        self.comm_session.writer = Mock()
```

Comment:
the test name should include the suffix invalid. Also I think you should create a test where the outcome is valid, if there's none yet

Reply:
Done